### PR TITLE
fixes #24603 - pulp-2.17 change in json

### PR DESCRIPTION
**Pull request by thomasmckay**
container image (docker) repo json is changing

### CODE REVIEW

File: app/models/katello/docker_manifest_list.rb [hunk 1]
```diff
@@ -25,7 +25,7 @@ def update_from_json(json)
       update_attributes(:schema_version => json[:schema_version],
                         :digest => json[:digest],
                         :downloaded => json[:downloaded],
-                        :docker_manifests => ::Katello::DockerManifest.where(:digest => json[:manifests])
+                        :docker_manifests => ::Katello::DockerManifest.where(:digest => json[:manifests].pluck(:digest))
```
Comment (johnpmitsch) [inline app/models/katello/docker_manifest_list.rb]: If `json` is actual JSON, will pluck work here? I thought it was an AR method only

Comment (thomasmckay) [reply app/models/katello/docker_manifest_list.rb]: @johnpmitsch - It works, which surprised me, and is very readable so I went that way.

**Comment by thomasmckay** [comment]
@johnpmitsch - is this ready for merge?

**Comment by johnpmitsch** [comment]
@thomasmckay I haven't tested out actual functionality, but will do that soon. Do you have general steps to test?

**Pull request merged**